README.md

```diff
@@ -13,38 +13,227 @@
+        ports:
+          - target: 3333
+            published: 3333
+            protocol: tcp
+            mode: host
+          - target: 8080
+            published: 3380
+            protocol: tcp
+            mode: host
+        secrets:
+          - source: config_json
+            target: config.json
+          - source: admin_fullchain_pem
+            target: admin_fullchain.pem
+          - source: admin_privkey_pem
+            target: admin_privkey.pem
+          - source: phish_fullchain_pem
+            target: phish_fullchain.pem
+          - source: phish_privkey_pem
+            target: phish_privkey.pem
+    ```
 
-    docker pull cisagov/gophish
+## Updating your container ##
 
-Or build `cisagov/gophish` from source:
+### Docker Compose ###
 
-    git clone https://github.com/cisagov/gophish-docker.git
-    cd gophish-docker
-    docker-compose build --build-arg VERSION=0.0.1
+1. Pull the new image from Docker Hub:
+
+    ```console
+    docker-compose pull
+    ```
+
+1. Recreate the running container by following the [previous instructions](#running-with-docker-compose):
+
+    ```console
+    docker-compose up --detach
+    ```
+
+### Docker ###
+
+1. Stop the running container:
+
+    ```console
+    docker stop <container_id>
+    ```
+
+1. Pull the new image:
+
+    ```console
+    docker pull cisagov/gophish:0.0.1
+    ```
+
+1. Recreate and run the container by following the [previous instructions](#running-with-docker).
+
+## Image tags ##
+
+The images of this container are tagged with [semantic
+versions](https://semver.org) of the underlying gophish project that they
+containerize.  It is recommended that most users use a version tag (e.g.
+`:0.0.1`).
+
+| Image:tag | Description |
+|-----------|-------------|
+|`cisagov/gophish:1.2.3`| An exact release version. |
+|`cisagov/gophish:1.2`| The most recent release matching the major and minor version numbers. |
+|`cisagov/gophish:1`| The most recent release matching the major version number. |
+|`cisagov/gophish:edge` | The most recent image built from a merge into the `develop` branch of this repository. |
+|`cisagov/gophish:nightly` | A nightly build of the `develop` branch of this repository. |
+|`cisagov/gophish:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |
+
+See the [tags tab](https://hub.docker.com/r/cisagov/gophish/tags) on Docker
+Hub for a list of all the supported tags.
+
+## Volumes ##
+
+There are no volumes.
 
-### Run ###
+<!-- | Mount point | Purpose        | -->
+<!-- |-------------|----------------| -->
+<!-- | `/var/log`  |  Log storage   | -->
 
-    docker-compose run --rm gophish
+## Environment variables ##
 
-## Ports ##
+### Required ###
 
-This container exposes the following ports:
+There are no required environment variables.
 
-- 3333: `admin server`
-- 8080: `phish server`
```

Comment:
The list of exposed ports is getting lost here, namely because we don't have that section in the skeleton.  Take a look at what I did [here](https://github.com/cisagov/postfix-docker/tree/lineage/skeleton#ports) and let me know what you think.

Reply:
That looks good to me, but we should add such a section to [cisagov/skeleton-docker](https://github.com/cisagov/skeleton-docker).  I can do that since I am creating another PR in that repo anyway.

See commit a21abdd.

Reply:
Perfect, thanks!

Reply:
I'm leaving this conversation unresolved as a reminder to wait for the upcoming skeleton-docker PR to flow in here before we merge this one.

Reply:
PR created - see cisagov/skeleton-docker#61